apps/src/lib/levelbuilder/unit-editor/UnitEditor.jsx

```diff
@@ -681,47 +682,41 @@ class UnitEditor extends React.Component {
         </CollapsibleEditorSection>
 
         <CollapsibleEditorSection title="Lesson Settings">
-          <label>
-            Use Code Studio Lesson Plans
-            <input
-              type="checkbox"
-              checked={
-                this.props.isMigrated && this.state.useCodeStudioLessonPlans
-              }
-              style={styles.checkbox}
-              onChange={() =>
-                this.setState({
-                  useCodeStudioLessonPlans: !this.state.useCodeStudioLessonPlans
-                })
-              }
-              disabled={!this.props.isMigrated}
-            />
-            <HelpTip>
-              {!this.props.isMigrated && (
-                <p>this option is only available for migrated scripts.</p>
-              )}
-              {this.props.isMigrated && (
+          {this.props.isMigrated && this.props.initialUseLegacyLessonPlans && (
+            <label>
+              <Button
+                text={'Use Code Studio Lesson Plans'}
```

Comment:
Is it worth adding a confirmation dialog here as we expect this to be an action that can't be reversed?

Reply:
![Screen Shot 2021-09-17 at 11 34 39 AM](https://user-images.githubusercontent.com/8001765/133837660-32eeeaa6-5c83-4bdf-9010-34189a386345.png)

good enough? also... after implementing this I totally accidentally clicked this button while scrolling up and down the edit page 😂

Reply:
haha I think that's good!